File: AirportManagement.API/AirportManagement.API/Data/Models/BoardingPass.cs
using System;
namespace AirportManagement.API.Models
{
	public class BoardingPass
	{
		public int Id { get; set; }

		public string QRCode { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime UpdatedAt { get; set; }

		public int BookingId { get; set; }

		public Booking Booking { get; set; }

	}
}

File: AirportManagement.SPA/src/app/_models/BoardingPass.ts
import { Booking } from "./Booking";

export interface BoardingPass {
    Id: number;
    QRCode: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    BookingId: number;
    Booking: Booking;
}

export interface BoardingPassForPassanger {
    qrCode: string,
    createdAt: Date;
    updatedAt: Date;
}